.github/workflows/evaluate.yml

```diff
@@ -124,7 +126,7 @@ jobs:
         run: |
           git clone --depth=1 https://github.com/open-compass/opencompass.git
           cd opencompass
-          python3 -m pip install --user -e .
+          python3 -m pip install opencompass==0.2.5
```

Comment:
Do we still have to `git clone` opencompass when it is installed through `pip install`?

Reply:
Can we remove the version limitation on opencompass?

Reply:
> Can we remove the version limitation on opencompass?

OC latest code use scikit_learn==1.5.0 which only support python>=3.9. Test image use python=3.8. So I need to fix the version.

Reply:
> Do we still have to `git clone` opencompass when it is installed through `pip install`?

need use model and dateset config files in OC, we still need download opencompass code

Reply:
I see. I am going to put upgrading python in docker image into my TODO list